### ISSUE TITLE
BUG: checkbox mangler tilgjengelig navn

### ISSUE THREAD

**Issue by hegeaal** [Informasjonsforvaltning/fdk-portal#1592]
## 🐛 Bug rapport

Fra pa11y: 

> This checkboxinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.

> This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.

Checkbox'ene i markdown får accessibility feil. Man finner feilen i datasettet GeoTest endret under "vis hele beskrivelsen".  Se bilde nr. 2. Feilen kan også finnes med WAVE. 

<img width="862" alt="image" src="https://user-images.githubusercontent.com/48259307/225305562-574d0a5d-d4a0-4548-b451-485511ff0d0e.png">

<img width="1019" alt="image" src="https://user-images.githubusercontent.com/48259307/225305520-d0bec246-0eca-4d4a-828d-b3f16b41c3c2.png">

<img width="604" alt="Screenshot 2023-03-15 at 13 14 54" src="https://user-images.githubusercontent.com/48259307/225306240-473f84a1-1a3e-4c3d-80a0-b5a907bdc2bc.png">

http://localhost:3001/datasets

Løsningsforslag: Tror man kan bruke `role='presentation'` i MarkDown komponenten i `src/components/markdown/index.tsx`.
